client/src/components/CreateJob/createjob.js

```diff
@@ -48,51 +52,91 @@ class CreateJob extends Component {
         description: this.state.description,
         budget: this.state.budget
       })
-        // .then(res => this.loadJobs())
+        .then(res => this.loadJobs())
         .catch(err => console.log(err));
     }
   };
 
   render() {
-    return (
-      <Container fluid>
-        <Row>
-          <Col size="md-6">
-            <Jumbotron>
-              <h1>Create Job</h1>
-            </Jumbotron>
-            <form>
-              <Input
-                type="text"
-                value={this.state.title}
-                onChange={this.handleInputChange}
-                name="title"
-                placeholder="Title (required)"
-              />
-              <TextArea
-                value={this.state.description}
-                onChange={this.handleInputChange}
-                name="description"
-                placeholder="Description (required)"
-              />
-              <Input
-                type="number"
-                value={this.state.budget}
-                onChange={this.handleInputChange}
-                name="budget"
-                placeholder="Budget (Optional)"
-              />
-              <FormBtn
-                disabled={!(this.state.description && this.state.title)}
-                onClick={this.handleFormSubmit}
-              >
-                Submit Job
-              </FormBtn>
-            </form>
-          </Col>
-        </Row>
-      </Container>
-    );
+    if (!this.props.loggedIn) {
+      return <Redirect to="/login" />;
+    } else {
+      return (
+        <div className="container">
```

Comment:
see if this guy was another component and you guys extracted out this whole mess it along with its other nested components it would be easier to follow. There is like this if statement with nested terenary operators within the component and it's hard to follow and debug in the future.